sql-plugin/src/main/scala/com/nvidia/spark/rapids/Plugin.scala

```diff
@@ -128,6 +130,76 @@ class RapidsExecutorPlugin extends ExecutorPlugin with Logging {
         ShimLoader.setSparkShimProviderClass(conf.shimsProviderOverride.get)
       }
 
+      try {
+        // Compare if the cudf version mentioned in the classpath is equal to the version which
+        // plugin expects. If there is a version mismatch, throw error. This check can be disabled
+        // by setting this config spark.rapids.cudfVersionOverride=true
+        val cudfPropertiesFileName = "cudf-java-version-info.properties"
+        val pluginPropertiesFileName = "rapids4spark-version-info.properties"
+
+        val props = new Properties
+        val cudfClassLoader = classOf[ai.rapids.cudf.ColumnVector].getClassLoader
```

Comment:
I don't see the need to use two different classloaders for the two different property files, as this class is able to reference cudf classes just fine.  Therefore we should be able to just use a single classloader like this:
```suggestion
        val classLoader = classOf[RapidsExecutorPlugin].getClassLoader
```